ethsigner/commandline/src/test-support/java/tech/pegasys/ethsigner/CmdlineHelpers.java

```diff
@@ -25,10 +25,7 @@ public static String validBaseCommandOptions() {
         + "--tls-keystore-file=./keystore.pfx "
         + "--tls-keystore-password-file=./keystore.passwd "
         + "--tls-known-clients-file=./known_clients "
-        + "--tls-allow-ca-clients "
-        + "--downstream-http-tls-keystore-file=./client_cert.pfx "
-        + "--downstream-http-tls-keystore-password-file=./client_cert.passwd "
-        + "--downstream-http-tls-known-servers-file=./knownServers.txt ";
+        + "--tls-allow-ca-clients ";
```

Comment:
it maybe worth leaving these things in, and having a fully-formed commandline as the default start point, then remove bits you don't need for explicit tests.

Reply:
done. test cases have been updated.